pyproject.toml

```diff
@@ -16,18 +16,18 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.8.0"
-pandas = "^1.5.3"
+pandas = "^2.1.1"
```

Comment:
@cebasfu93 could you replace this with ">=1.5.3" instead of bumping to `2` i don't know how many users still might be using pandas `1`

This package makes minimal usage of pandas features, so I think we want to relax constraints as much as possible to accommodate more users

Reply:
Good point :) 
Thanks for the quick review!